deploy/awstools/awstools.py

```diff
@@ -472,6 +481,7 @@ def main(args):
     parser.add_argument("--block_devices", type=yaml.safe_load, default=run_block_device_dict(), help="List of dicts with block device information. Used by \'launch\'.")
     parser.add_argument("--tags", type=yaml.safe_load, default=run_tag_dict(), help="Dict of tags to add to instances. Used by \'launch\'.")
     parser.add_argument("--filters", type=yaml.safe_load, default=run_filters_list_dict(), help="List of dicts used to filter instances. Used by \'terminate\'.")
+    parser.add_argument("--user_data_file", default=None, help="File path to use as user data. Used by \'launch\'.")
```

Comment:
You should probably say "run on initialization" in a parenthetical or something to describe what user data is.